### PR TITLE
Update /about and /covid, and other page styles

### PR DESCRIPTION
**Pull request by adamdehaven**
Updates the copy and layout for:

- `/about`
- `/covid`

Updates the layout & styles for:

- `/on-demand`
- `/partners`

### CODE REVIEW

File: src/components/page-covid/page-covid.tsx [hunk 1]
```diff
@@ -1,67 +1,95 @@
...
+                  </a>{" "}
+                  with delivery drivers and restaurants, but it’s up to each person to consider their own personal risks.
+                </p>
+                <hr />
+                <h3 class="instructions-subheader">Are you a restaurant or helping distribute food?</h3>
+                <stencil-route-link url="/guidelines" anchorClass="button is-teal">
+                  View our guidelines for distributors
+                </stencil-route-link>
+                <p>
+                  <stencil-route-link url="/insstructions" anchorClass="has-text-teal">
```
Comment (mojowen) [inline src/components/page-covid/page-covid.tsx:87]: /insstructions/instructions/

**Comment by mojowen** [comment]
Could you also update this to read:

```
  If you have any questions, you can call our hotline at <a href="tel:971-407-1829">971-407-1829</a>
```

image

**Review by mojowen — APPROVED** [review]
Looks great!

**Comment by mojowen** [comment]
Feel free to merge when you're ready - thank you for your help!

File: src/components/page-covid/page-covid.tsx (hunk 1 above)
Comment (adamdehaven) [reply src/components/page-covid/page-covid.tsx]: Got it.

**Comment by adamdehaven** [comment]
I also added the country code in the tel (just in case):
`<a href="tel:+1-971-407-1829">971-407-1829</a>`

**Pull request merged**